integrate_and_upload.py

```
@@ -0,0 +1,198 @@
+
+    all_pins = []
+    all_files = [os.path.join(path, name) for path, subdirs, files in os.walk(DOWNLOAD_DIR) for name in files if name.endswith('.csv')]
+
+    for filepath in all_files:
+        try:
+            df = pd.read_csv(filepath, encoding='utf-8-sig')
+            
+            #【修正点】NaN（非数）をNone（null）に置換して、JSONエラーを防ぐ
+            df = df.replace({np.nan: None}) 
+
+            for index, row in df.iterrows():
```

Ruff (0.11.9)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)